hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -131,6 +177,13 @@ private DifferSnapshotInfo getDSIFromSI(SnapshotInfo snapshotInfo,
         getTablePrefixes(snapshotOMMM, volumeName, bucketName));
   }
 
+  private Set<String> getSSTFileListForSnapshot(OmSnapshot snapshot,
+          List<String> tablesToLookUp) throws RocksDBException {
+    return RdbUtil.getSSTFilesForComparison(snapshot
+            .getMetadataManager().getStore().getDbLocation()
+                    .getPath(), tablesToLookUp);
+  }
+
   @SuppressWarnings("checkstyle:methodlength")
   public SnapshotDiffReport getSnapshotDiffReport(final String volume,
```

Comment:
Need to have more test cases to cover the branch additions in this method.

Reply:
This could be a separate PR. [HDDS-7798](https://github.com/apache/ozone/pull/4187)